src/complex/Core/Application.hpp

```diff
@@ -118,6 +118,18 @@ class COMPLEX_EXPORT Application
    */
   std::filesystem::path getCurrentDir() const;
 
+  /**
+   * @brief Returns the Complex filter UUID [v4] from the SIMPL filter UUID [v5]
+   * @return std::optional<Uuid>
+   */
+  std::optional<Uuid> getComplexUuid(const Uuid& simplUuid);
+
+  /**
+   * @brief Returns the SIMPL filter UUID(s) [v5] from the Complex filter UUID [v4]
+   * @return std::optional<std::vector<Uuid>>
+   */
+  std::optional<std::vector<Uuid>> getSimplUuid(const Uuid& complexUuid);
```

Comment:
Why does this return a vector of uuids?

Reply:
There are multiple dream3d filters that map to a single complex filter